sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpEventBatch.cs

```diff
@@ -123,43 +126,26 @@ public override bool TryAdd(EventData eventData)
             Argument.AssertNotNull(eventData, nameof(eventData));
             Argument.AssertNotDisposed(_disposed, nameof(EventDataBatch));
 
-            // Reserve space for producer-owned fields that correspond to special
-            // features, if enabled.
-
-            if ((ActiveFeatures & TransportProducerFeatures.IdempotentPublishing) != 0)
```

Comment:
I forgot to add idempotent property reservation back in after collapsing back to a single batch.     Will fix.